roles/ceph-osd/tasks/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: delete default 'rbd' pool
+  command: ceph osd pool delete rbd rbd --yes-i-really-really-mean-it
```

Comment:
Looks like no, but we should maybe add the code to that module to support deleting a pool...